src/date-range-picker/interfaces.ts

```diff
@@ -264,6 +264,12 @@ export namespace DateRangePickerProps {
      */
     ariaDescribedby?: string;
 
+    /**
+     * Label of the mode selection group. In the standard view it adds `aria-label` on the group of segments.
```

Comment:
Slight grammar tweak: "Label of the mode selection group. In the standard view, it adds 'aria-label' to the group of segments."